assets/android/kotlin/DarkDesignTokens.kt

```diff
@@ -167,118 +167,118 @@ object DarkDesignTokens {
   val colorYellow800 = Color(0xffa76300)
   val colorYellow900 = Color(0xff773800)
   val fontBodyLgMedium = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W500
-lineHeight = 22.dp
-fontSize = 16.dp
-letterSpacing = 0.015625.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W500,
+lineHeight = 22.sp,
+fontSize = 16.sp,
+letterSpacing = 0.015625.em,
 )
   val fontBodyLgRegular = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W400
-lineHeight = 22.dp
-fontSize = 16.dp
-letterSpacing = 0.015625.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W400,
+lineHeight = 22.sp,
+fontSize = 16.sp,
+letterSpacing = 0.015625.em,
 )
   val fontBodyMdMedium = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W500
-lineHeight = 20.dp
-fontSize = 14.dp
-letterSpacing = 0.017857.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W500,
+lineHeight = 20.sp,
+fontSize = 14.sp,
+letterSpacing = 0.017857.em,
 )
   val fontBodyMdRegular = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W400
-lineHeight = 20.dp
-fontSize = 14.dp
-letterSpacing = 0.017857.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W400,
+lineHeight = 20.sp,
+fontSize = 14.sp,
+letterSpacing = 0.017857.em,
 )
   val fontBodySmMedium = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W500
-lineHeight = 17.dp
-fontSize = 12.dp
-letterSpacing = 0.033333.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W500,
+lineHeight = 17.sp,
+fontSize = 12.sp,
+letterSpacing = 0.033333.em,
 )
   val fontBodySmRegular = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W400
-lineHeight = 17.dp
-fontSize = 12.dp
-letterSpacing = 0.033333.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W400,
+lineHeight = 17.sp,
+fontSize = 12.sp,
+letterSpacing = 0.033333.em,
 )
   val fontBodyXsMedium = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W500
-lineHeight = 15.dp
-fontSize = 11.dp
-letterSpacing = 0.045454.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W500,
+lineHeight = 15.sp,
+fontSize = 11.sp,
+letterSpacing = 0.045454.em,
 )
   val fontBodyXsRegular = TextStyle(
-fontFamily = "Roboto"
-fontWeight = FontWeight.W400
-lineHeight = 15.dp
-fontSize = 11.dp
-letterSpacing = 0.045454.em
+fontFamily = FontFamily.Default,
+fontWeight = FontWeight.W400,
+lineHeight = 15.sp,
+fontSize = 11.sp,
+letterSpacing = 0.045454.em,
 )
   val fontFamilyMono = "Roboto Mono"
   val fontFamilySans = "Roboto"
```

Comment:
I didn't notice these before. They aren't in use, but I guess they should be `FontFamily` values too?

By their names I guess they should be:
```kotlin
val fontFamilyMono = FontFamily.Monospace
val fontFamilySans = FontFamily.SansSerif
```

And maybe be reused for some future `TextStyles`? If that's the case, they should probably be at the top? So objects declared below them can access them.

Reply:
I think we are going to filter those values out from the output as there's no reason to use them.